lib/apiservers/portlayer/swagger.json

```diff
@@ -1748,6 +1748,63 @@
 				}
 			}
 		},
+		"/containers/{handle}/name": {
+			"put": {
```

Comment:
Agree with @vburenin on docker not really using REST. I would however go with PUT; PATCH is not really very widely used.